src/Pages/Home.js

```diff
@@ -79,7 +83,8 @@ class Home extends Component {
           text={this.props.t('moreArticles')}
         />
         {home.ourStory &&
-          home.ourStory.length && <OurStory fact={home.ourStory[0]} />}
+          home.ourStory.value &&
+          home.ourStory.length && <OurStory fact={home.ourStory.value[0]} />}
```

Comment:
Is length check/unwrap necessary here?

Reply:
Hi @makma, It is actually not, I have removed it.